src/orquestra/integrations/custatevec/simulator/custatevec.py

```diff
@@ -0,0 +1,63 @@
+################################################################################
+# © Copyright 2021-2022 Zapata Computing Inc.
+################################################################################
+
+import sys
+import warnings
+from typing import Optional
+
+import cirq
+
+from ...cirq.simulator.qsim_simulator import QSimSimulator
+
+try:
+    import qsimcirq  # type: ignore
+except ModuleNotFoundError:
+    warnings.warn("qsimcirq is not imported")
+
+
+class CustatevecSimulator(QSimSimulator):
+
+    """CustatevecSimulator is qsimcirq simulator that
```

Comment:
I think it's worth adding to the docstring that in order for this to work you need to meet several conditions, such as having NVIDIAs GPU and all the things actually installed on your machine.